Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -1166,9 +1167,12 @@ extension DocumentSnapshot {
     of range: Range<Position>,
```

Comment:
Could you rename this function to `absolutePositionRange` because it no longer returns a `ByteSourceRange`?